src/coreComponents/mesh/generators/WellBlock.hpp

```diff
@@ -208,15 +159,15 @@ class WellBlock : public WellBlockABC
   /// Radius area of the well (assumed to be valid for the entire well)
   real64 m_radius;
 
+  /// Name of the corresponding well region
+  string m_name;
```

Comment:
I think that this conflicts with the `m_name` from `Group`.

Reply:
Removed, thanks